### PR TITLE
MTM-55444: edge-databroker: added manual uninstall instructions

### PR DESCRIPTION
**Pull request by gapa-c8y**
Requested by the edge team. 

Possible problem: Not sure why I can't change the title in index.html (and if I should).
It seemed wrong to be duplicated, and to be the parent of this uninstall guide.

@bbyreddy , please add other Edge-team reviewers to wish (I couldn't spot others).

I'm not expecting any back _or forward_ ports of this

### CODE REVIEW

File: content/edge/databroker-bundle/databroker-edge-uninstall.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
+---
+weight: 50
+title: Uninstalling the Messaging Service and the microservice-based data broker on Edge
+layout: redirect
+---
+
+The bundle for the Messaging Service and data broker microservice does not have an uninstall command. 
+If a mistake is made, or and installation fails, the bundle components must be manually uninstalled.
+Once manually uninstalled, installation can be reattempted.
```
Comment (MWindel) [inline content/edge/databroker-bundle/databroker-edge-uninstall.md:9]:
```suggestion
Once manually uninstalled, the installation can be reattempted.
```

File: content/edge/databroker-bundle/databroker-edge-uninstall.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+#### To manually uninstall:
+1. Login to the edge machine using SSH.
```
Comment (MWindel) [inline content/edge/databroker-bundle/databroker-edge-uninstall.md:12]:
```suggestion
1. Login to the {{< product-c8y-iot >}}  Edge machine using SSH.
```

Comment (MWindel) [reply content/edge/databroker-bundle/databroker-edge-uninstall.md:12]: Is it really "Edge machine"? What do you mean by that?

Comment (gapa-c8y) [reply content/edge/databroker-bundle/databroker-edge-uninstall.md:12]: I look through existing docs and see what terminology is used elsewhere, and then try again. Thanks

Comment (gapa-c8y) [reply content/edge/databroker-bundle/databroker-edge-uninstall.md]: I've had a go (it appears it should be simply referred to C8Y IOT Edge). Please let me know if you still don;t like it. 

Maybe I should provide ssh login code example too (maybe that's going to far and patronizing the reader).

**Comment by BeateRixen** [comment]
@gapa-c8y I assume that this will be added to the develop branch in a separate PR after approval?

**Comment by gapa-c8y** [comment]
> @gapa-c8y I assume that this will be added to the develop branch in a separate PR after approval?

Maybe I should redo this for develop instead. It is for 1017 and the process for bundling this stuff for Edge is expected to be totally different in the future. However, that hasn't happened yet so we should be prepared for it to not happen. I should maybe recreate the PR and do it in develop (and backport to 1018 and 1017).

If you agree, I'll close this PR

**Comment by BeateRixen** [comment]
> > @gapa-c8y I assume that this will be added to the develop branch in a separate PR after approval?
> 
> Maybe I should redo this for develop instead. It is for 1017 and the process for bundling this stuff for Edge is expected to be totally different in the future. However, that hasn't happened yet so we should be prepared for it to not happen. I should maybe recreate the PR and do it in develop (and backport to 1018 and 1017).
> 
> If you agree, I'll close this PR

Please note that you cannot easily backport to 10.17 and 10.18 so we need 2 PRs anyway.

**Comment by sagscmi** [comment]
> > @gapa-c8y I assume that this will be added to the develop branch in a separate PR after approval?
> 
> Maybe I should redo this for develop instead. It is for 1017 and the process for bundling this stuff for Edge is expected to be totally different in the future. However, that hasn't happened yet so we should be prepared for it to not happen. I should maybe recreate the PR and do it in develop (and backport to 1018 and 1017).
> 
> If you agree, I'll close this PR

I would check with @bbyreddy before doing anything, e.g. I'm pretty certain there will never be a 10.18 Edge release.  Probably does make sense to have it on develop though in case the new K8S deployment plan doesn't work out.

**Comment by gapa-c8y** [comment]
> > > @gapa-c8y I assume that this will be added to the develop branch in a separate PR after approval?
> > 
> > 
> > Maybe I should redo this for develop instead. It is for 1017 and the process for bundling this stuff for Edge is expected to be totally different in the future. However, that hasn't happened yet so we should be prepared for it to not happen. I should maybe recreate the PR and do it in develop (and backport to 1018 and 1017).
> > If you agree, I'll close this PR
> 
> Please note that you cannot easily backport to 10.17 and 10.18 so we need 2 PRs anyway.

Ok. I can change this to a 1018 PR for backporting to 1017 (just to be more normal/conventional).
And create a PR for develop. 

When Edge do change this stuff drastically as expected, the rework of the the Edge guides won't be forgotten. It seems safer to make sure it is in all branches until actually changed.

**Comment by gapa-c8y** [comment]
> > > @gapa-c8y I assume that this will be added to the develop branch in a separate PR after approval?
> > 
> > 
> > Maybe I should redo this for develop instead. It is for 1017 and the process for bundling this stuff for Edge is expected to be totally different in the future. However, that hasn't happened yet so we should be prepared for it to not happen. I should maybe recreate the PR and do it in develop (and backport to 1018 and 1017).
> > If you agree, I'll close this PR
> 
> I would check with @bbyreddy before doing anything, e.g. I'm pretty certain there will never be a 10.18 Edge release. Probably does make sense to have it on develop though in case the new K8S deployment plan doesn't work out.

Sorry @sagscmi, I missed this when responding myself. It shouldn't increase the effort mush in any case as there won't be any conflicts (new file)

**Pull request closed without merging**